CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 ## Unreleased
 
+## [v2.2.1] - 2024-09-11
```

Comment:
Wouldn't this be a major version release as it's a breaking change dropping Python 3.8 support?

Reply:
hmmm I think that's probably correct. I was thinking that I don't actually expect it to break 3.8 itself (I did test that it works with 3.8)